libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/hybrid/MapboxHybridRouter.kt

```diff
@@ -28,12 +34,38 @@ class MapboxHybridRouter(
     networkStatusService: NetworkStatusService
 ) : Router {
 
+    constructor(
```

Comment:
never used, may be avoid create additional constructor 🤔

Reply:
Oh, I got it. resolved

Reply:
This constructor is used when injecting the module in the `MapboxNavigation`.